.ci/jenkins/dsl/jobs.groovy

```diff
@@ -54,6 +54,10 @@ Map getMultijobPRConfig(boolean isNative = false) {
             ]
         ]
     ]
+    if (isNative) { // Optawebs should not be used in native.
+        jobConfig.jobs.retainAll { !it.id.startsWith('optaweb') }
```

Comment:
My IDE is giving me some strange warning that this is not supported on `ArrayList`. (Even though the Groovy Javadoc clearly says that it is.) Any advice?

Reply:
well Jenkins is using quite an old version of Groovy ...
but seems it is in: https://docs.groovy-lang.org/1.8.9/html/groovy-jdk/java/util/Collection.html#retainAll(java.lang.Object[])